pkg/alerts/plugins_conditions.go

```diff
@@ -107,9 +107,9 @@ func (a *Alerts) UpdatePluginsCondition(condition PluginsCondition) (*PluginsCon
 // DeletePluginsCondition deletes a plugin alert condition.
 func (a *Alerts) DeletePluginsCondition(id int) (*PluginsCondition, error) {
 	resp := pluginConditionResponse{}
-	u := fmt.Sprintf("/alerts_plugins_conditions/%d.json", id)
+	url := fmt.Sprintf("/alerts_plugins_conditions/%d.json", id)
 
-	_, err := a.client.Delete(u, nil, &resp)
+	_, err := a.client.Delete(a.config.Region().RestURL(url), nil, &resp)
```

Comment:
For these kinds of calls, I was seeing the parts passed to the `RestURL()` method as "path".  Was the intent here to have something like `a.config.Region().RestURL("alerts_plugins_conditions", string(id), "json")` ?

Reply:
Yes, to enable something like:
```
a.config.Region().RestURL("alerts_plugins_conditions", strconv.Itoa(id)+".json")
```

Note using `+` as all strings passed in will be joined with `/` otherwise.

Reply:
Yeah, I like that.